src/screens/IndexScreen.js

```diff
@@ -322,6 +348,14 @@ export const IndexScreen = ({ navigation, route }) => {
               ]) &&
               !Object.prototype.hasOwnProperty.call(queryVariables, 'refetch');
 
+            // apply additional data if volunteer events should be presented and
+            // no category selection is made, because the category has nothing to do with
+            // volunteer data
+            const additionalData =
+              isCalendarWithVolunteerEvents && !hasCategoryFilterSelection
+                ? dataVolunteerEvents
+                : undefined;
```

Comment:
what happens if we use the `&&` operator instead of `?` here?

Reply:
`additionalData` would be boolean or array instead of undefined or array. but we types with `additional?: unknown[]` which corresponds to undefined or array.

Reply:
Thanks for the information. 👍